GOVERNANCE.md

```diff
@@ -0,0 +1,125 @@
+Academic members. All Platinum and Silver members must be current corporate
+members of the hosting foundation (e.g. Linux Foundation or Open Infrastructure
+Foundation) at any level to participate in the Diamond Bluff project.
+
+Platinum, Silver and Participating Academic Members shal be entitled to:
+
+* Participate in the Diamond Bluff Project memtings, initiatives, events and
+  any other activities.
+* Identify their company as a member or participant in the Diamond Bluff
+  project.
+* Platinum members shall be entitled to appoint a representative to the
+  Board of Directors of the Diamond Bluff project, as well as any future
+  committees established by the Board of Directors.
+
+## Structure
+
+At a high level, the Diamond Bluff project is governed by two main bodies which
+work together to achieve the project's goals:
+
+* A Board of Directors
+* A Technical Steering Committee
+
+### Board of Directors
+
+The Diamond Bluff project Board of Directors shall be composed as follows:
+
+* One representative from each Platinum member.
+* One Silver member representative elected annually by the Silver members for
+  every 5 Silver members, with at least one representative, and no more than 3.
+
+The Board of Directors shall have the following responsibilities:
+
+* Approve a budget directing the use of funds raised from all sources of
+  revenue.
+* Elect a Chair of the Diamond Bluff project to preside over meetings,
+  authorize expenditures approved by the budget and manage any day-to-day
+  operations, and vote on decisions or matters before the Board of Directors.
+
+The Board of Directors shall::
+
+* Demonstrate plans and the means to coordinate with the open source project’s
+  developer community, including on topics such as branding, logos, and other
+  collateral that will represent the community.
+* Engage in a professional manner consistent with maintaining a cohesive
+  community, while also maintaining the goodwill and esteem of the foundation
+  the Diamond Bluff Project is a member of, in the open source software
+  community.
+* Respect the rights of all trademark owners, including any branding and usage
+  guidelines.
+* Engage the foundation the project is a member of for all press and analyst
+  relations activities.
+* Upon request, provide information regarding project participation, including
+  information regarding attendance at project-sponsored events, to the
+  foundation the project is a member of.
+* Engage foundation the project is a member of for any websites directly for
+  the Diamond Buff Project; and operate under such rules and procedures as may
+  from time to time be approved by the Diamond Bluff Board of Directors and
+  confirmed by foundation the project is a part of.
+
+### Technical Steering Committee
+
+The Diamond Bluff Technical Steering Committee shall be composed as follows:
+
+* One appointed representative from each Platinum member
```

Comment:
So by this formulation, the majority of the members of the TOC would also be direct appointees, rather than representing the projects.  This makes the TOC not at all differentiated from the Board.  We need a structure that makes sure that we get a different group of people for the TOC; let me see if I can add some text to make that happen.

Reply:
Excellent and yes, I want the Technical Steering Committee (TSC) to be a bit more technical and less appointed, I can put some text to change that if you can't push to this PR.

Reply:
What's the largest number of projects we expect to have under DB?

Reply:
That's hard to predict, but this governance document is trying to take a first crack at how those might be formed and guided by the Technical Steering Committee. I can see 3 initial projects: A DPU/IPU infra project (think base OS and packaging), IPDK (table driver interface abstractions), and testing.

Reply:
I thought that we were going to adopt something along the lines of an appointed TSC member from each platinum member to start with, and have 1/3 rotate off each of the first 3 years, to make way for TSC members to be nominated and voted in on technical contribution merit. that way we have a TSC to begin with that will have some consistency over the beginning years, but end up with those of great technical merit in the end.

Reply:
I do not have a problem adding a maintainer from each subproject either.

Reply:
Yeah, that's what I recall too.

Reply:
The latest version is updated to reflect this discussion.